run_kraken.py

```diff
@@ -348,21 +350,18 @@ def main(cfg):
 
                         # Check for critical alerts when enabled
                         if check_critical_alerts:
-                            logging.info("Checking for critical alerts firing post choas")
-
-                            ##PROM
-                            query = r"""ALERTS{severity="critical"}"""
-                            end_time = datetime.datetime.now()
-                            critical_alerts = prometheus.process_query(
-                                query
-                            )
-                            critical_alerts_count = len(critical_alerts)
-                            if critical_alerts_count > 0:
-                                logging.error("Critical alerts are firing: %s", critical_alerts)
-                                logging.error("Please check, exiting")
+                            prometheus_plugin.critical_alerts(prometheus,
+                                                              summary,
+                                                              run_uuid,
+                                                              scenario_type,
+                                                              start_time,
+                                                              datetime.datetime.now())
+
+                            chaos_output.critical_alerts = summary
+                            if len(summary.post_chaos_alerts) > 0:
```

Comment:
below we do a check (~line 443) to exit 1 if there are alerts at the end of the run. Need the variable to be updated based on these changes. It is exiting with success currently even though I had a critical error firing 

```
2024-02-20 19:02:25,975 [INFO] telemetry collection disabled, skipping.
2024-02-20 19:02:25,975 [INFO] Successfully finished running Kraken. UUID for the run: 6ca05da4-81ac-49fd-8730-0c5005f4515c. Report generated at kraken.report. Exiting
```

Reply:
@chaitanyaenr told me to exit only if there were `post` scenario critical alerts so I made this change yesterday, if we wanna exit for any alert I can add the check on both of them.

Reply:
I got that, but I think we still need to check at the end of the run if the post chaos alerts were firing when we break on line 363 to tell krkn the run had the post scenario failures. Just need to change the variable on the check below to either summary.post_chaos_alerts or chaos_output.critical_alerts.post_chaos_alerts

https://github.com/tsebastiani/krkn/blob/bf3a652f895c4ab1c62ed7d024719624c3734d6d/run_kraken.py#L443-L445

Reply:
actually think we need to just set critical_alerts_count in the if statement that this comment is on so that it'll be set if alerts are turned off but set if they are on